packages/preview/slydst/0.1.0/typst.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "slydst"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Gaspard Lambrechts"]
+license = "MIT"
+description = "Create simple static slides using standard headings"
```

Comment:
could you add an `exclude = ["svg", "example.typ"]` list to reduce the package size?

Reply:
Done, but isn't it a problem to include the `README.md` without the images it contains?

Reply:
I think it's okay. It will very rarely be viewed in the archive, but I still think it's good to package it along since it's small and still usable, just degraded. The examples, however, would bloat the archive.